stylesheets/styles.css

```diff
@@ -190,34 +189,53 @@ hr {
 }
 
 footer {
-  width: 270px;
-  float: left;
-  position: fixed;
-  bottom: 50px;
+  max-width: 860px;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+
+.donation-box {
+  border: 4px solid #4bafff;
+  padding: 65px 0;
+  margin-bottom: 50px;
+}
+
+.donation-box p {
+  font-size: 20px;
+}
+
+.donation-box a {
+  background-color: #4bafff;
+  color: #fff;
+  font-weight: bold;
+  font-size: 18px;
+  padding: 10px 13px;
+  border-radius: 5px;
 }
 
 @media print, screen and (max-width: 960px) {
 
   div.wrapper {
     width: auto;
     margin: 0;
+    flex-direction: column;
+  }
+
+  header {
+    width: 100%;
+    text-align: center;
   }
 
-  header, section, footer {
+  section, footer {
     float: none;
```

Comment:
@KimberleyCook do we still need this code now that we are not floating anything?

Reply:
probably yes, and there is likely more I could delete too 😂